### ISSUE TITLE
Breaking changes with new package installation

### ISSUE THREAD

**Issue by mickael-btc** [massalabs/create-react-app-massa#9]
## Description
When running npm install on an older version of the project with an outdated package-lock.json, the dependencies are installed correctly and the app runs without issue. However, when attempting to install new packages or update existing ones, there are breaking changes that prevent the app from running.

## Steps to reproduce
- Start with an older version of the project that has an outdated `package-lock.json`
- Run `npm install` and observe that the app runs without issue
- Attempt to install new packages or update existing ones by running a fresh `npm`, `yarn` or `pnpm install`
- Observe that the app is now unrunnable due to breaking changes

## Expected behavior
The app should be able to run correctly after installing new packages or updating existing ones.

## Actual behavior
The app is unrunnable due to breaking changes.

## Additional information
I got this error while running the project with a fresh install
```txt
Compiled with problems:X

ERROR in ./node_modules/ws/lib/permessage-deflate.js 3:13-28

Module not found: Error: Can't resolve 'zlib' in '/home/mickael/tmp/create-react-app-massa/node_modules/ws/lib'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "zlib": require.resolve("browserify-zlib") }'
	- install 'browserify-zlib'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "zlib": false }


ERROR in ./node_modules/ws/lib/sender.js 5:12-26

Module not found: Error: Can't resolve 'net' in '/home/mickael/tmp/create-react-app-massa/node_modules/ws/lib'


ERROR in ./node_modules/ws/lib/sender.js 6:12-26

Module not found: Error: Can't resolve 'tls' in '/home/mickael/tmp/create-react-app-massa/node_modules/ws/lib'


ERROR in ./node_modules/ws/lib/websocket-server.js 8:12-26

Module not found: Error: Can't resolve 'net' in '/home/mickael/tmp/create-react-app-massa/node_modules/ws/lib'


ERROR in ./node_modules/ws/lib/websocket-server.js 9:12-26

Module not found: Error: Can't resolve 'tls' in '/home/mickael/tmp/create-react-app-massa/node_modules/ws/lib'


ERROR in ./node_modules/ws/lib/websocket.js 8:12-26

Module not found: Error: Can't resolve 'net' in '/home/mickael/tmp/create-react-app-massa/node_modules/ws/lib'


ERROR in ./node_modules/ws/lib/websocket.js 9:12-26

Module not found: Error: Can't resolve 'tls' in '/home/mickael/tmp/create-react-app-massa/node_modules/ws/lib'
```

**Comment by BatiGencho** [comment]
Hey, @mickael-btc I have just released the latest version of the create-react-app-massa with all the later packages and fixes. Please give it a try and let me know if any issues. If not, pls close this issue. Thanks.

**Comment by mickael-btc** [comment]
Hello,

I am now able to launch this demo when using pnpm. However it seems that there is a small bug on the main page:

```
Compiled with problems:X

ERROR in src/App.test.tsx:8:23

TS2339: Property 'toBeInTheDocument' does not exist on type 'JestMatchers<HTMLElement>'.
     6 |   render(<App />);
     7 |   const linkElement = screen.getByText(/learn react/i);
  >  8 |   expect(linkElement).toBeInTheDocument();
       |                       ^^^^^^^^^^^^^^^^^
     9 | });
    10 |

```

Otherwise the depedency seems to works well !

**Comment by BatiGencho** [comment]
@mickael-btc How do you get this ? Windows, Linux, or Mac ? I cannot see that. Did you run `npm run test` (as this is the standard test file) or ?

**Comment by mickael-btc** [comment]
To reproduce try this:
1. `git clone https://github.com/massalabs/create-react-app-massa.git`
2. `cd create-react-app-massa`
3. `pnpm install`
4. `npm run start`
5. you should be able to see this:
![image](https://user-images.githubusercontent.com/48947931/215785574-84345947-1b7e-4c3c-a7c9-dced9c7cc914.png)

**Comment by mickael-btc** [comment]
My config: WLS 2 on Windows 10 with pnpm 7.17.0

**Comment by BatiGencho** [comment]
@mickael-btc I have released a new version now  (1.3.1) with some upgrades. On my machine it works fine with npm and yarn and NodeJS take pretty much any version. I don't know much pnpm and am afraid I wont be able to fix the issue right away. I tried it but it seems to lead to an error but not the one you have on your machine. I would not recommend using pnpm, maybe just try npm or yarn if that is ok for you and let me know how it goes.
